### DISCUSSION TITLE
How long would a 🚀🧑‍🚀 flight between London and Sydney take?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#865] [Galaxy Trivia] [answered]
*according to recent research*

**Comment by Carol42** [comment, marked as answer]
New research shows passengers will be able to travel from London to Sydney in less than two hours within a decade by travelling through space.

The study, published in the journal Aerospace Medicine and Human Performance, said: “Commercial* suborbital space flights are now available for tourism and scientific research, and are ultimately anticipated to mature into extremely fast point-to-point travel — for example, London to Sydney in less than two hours.”

**Comment by peterbaccaro** [comment]
New research suggests that a flight from Sydney to London will take just **2 hours** by 2033, involving sub-orbital 'earth to earth' space flights—it's a journey that currently takes 22 hours.

https://www.forbes.com/sites/alexledsom/2023/05/16/2-hour-london-to-sydney-flights-by-2033-says-new-report
